### PR TITLE
Open cabal:// urls into the app

### LINKED ISSUE #74: Register cabal-desktop to handle cabal links
It looks like Electron has an API to register to handle custom protocols:

https://github.com/electron/electron/blob/master/docs/api/protocol.md

It would be great if `cabal-desktop` registered to handle `cable://<key>` urls. That would make sharing links to cabals *way* easier.

Perhaps this could be our first preference? 😉

### PR DESCRIPTION
**Pull request by nikolaiwarner**
fixes #74

This adds the feature for clicking a cabal:// url which will open the app and either switch to that cabal or attempt to add it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,16 +16,16 @@ https://github.com/cabal-club/cabal-desktop/releases/
 $ git clone https://github.com/cabal-club/cabal-desktop
 $ cd cabal-desktop
 
-$ npm install             # install dependencies
-$ npm start               # start the application
+$ yarn install             # install dependencies
```
Comment (noffle) [inline README.md]: Is yarn a requirement now?

Comment (noffle) [reply README.md]: If not, I think more people know/have npm than yarn, so it might be worth keeping npm here even just for sake of clarity.

Comment (nikolaiwarner) [reply README.md]: npm's fine with me! @d4tocchini made some improvements recently that let yarn keep things leaner... but that may only be for doing a production build anyhow. I'll change it back.

File: README.md [hunk 2]
```diff
@@ -16,16 +16,16 @@ https://github.com/cabal-club/cabal-desktop/releases/
...
+$ npm start               # start the application
 ```
 
 ## Distribute
 
 build for current platform:
 
 ```
-$ yarn run dist
+$ npm run dist
```
Comment (ralphtheninja) [inline README.md]: :heart:

**Pull request merged**